src/components/asset-list/RecyclerAssetList2/profile-header/ProfileActionButtonsRow.tsx

```diff
@@ -0,0 +1,419 @@
+import Clipboard from '@react-native-community/clipboard';
+import lang from 'i18n-js';
+// @ts-expect-error - JS module
+import { IS_TESTING } from 'react-native-dotenv';
+import * as React from 'react';
```

Comment:
```suggestion
import React from 'react';
```

Reply:
I like importing related things on a namespace, personally. But that's entirely my opinion 😛

Reply:
Thought importing on a namespace (i.e. `import * as React from 'react'` instead of `import React from 'react') was the way to go in TS. Remember seeing something on why this is the case, but forgot.

Should this be in our lint rules?